docs/core/getting-started/install/mac.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Install for macOS 10.11 or higher
+description: Install for macOS 10.11 or higher
+keywords: .NET, .NET Core, C#, Getting Started, Acquisition, Cross Platform, macOS
+author: kendrahavens
+ms.author: kendrahavens
+ms.date: 12/19/2016
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+ms.assetid: 
+---
+
+# Install for macOS 10.11 or higher
+
+## 1. Install pre-requisites
+- In order to use .NET Core, you first need to install the latest version of OpenSSL. The easiest way to get this is from [Homebrew](http://brew.sh/).
+- After installing brew, do the following:
+```
+brew update
+brew install openssl
+mkdir -p /usr/local/lib
+ln -s /usr/local/opt/openssl/lib/libcrypto.1.0.0.dylib /usr/local/lib/
+ln -s /usr/local/opt/openssl/lib/libssl.1.0.0.dylib /usr/local/lib/
+```
+[!TIP] [Video: Install .NET Core and Visual Studio Code on a Mac](https://sec.ch9.ms/ch9/91a8/38eb734a-ea5a-4e3d-92ee-15de701391a8/VSCodeMac_high.mp4)
```

Comment:
Right syntax for tips is (notice the greater then sign):
```markdown
> [!TIP]
tip text
```

same for notes as below